cl/donate/api_views.py

```diff
@@ -232,6 +243,28 @@ def _map_trigger_value(self, trigger_event: str) -> int:
 
         return trigger
 
+    def _map_payment_status_value(self, status: str) -> int:
```

Comment:
This method can be `staticmethod`

```suggestion
    @staticmethod
    def _map_payment_status_value(status: str) -> int:
```

Similar suggestion for:
`_get_membership_data`, `_map_trigger_value`, `_handle_membership_deletion`  and `_map_payment_status_value`